src/js/program/media-controller.js

```diff
@@ -121,7 +121,7 @@ export default class MediaController extends Eventable {
         });
         // Immediately set player state to buffering if these conditions are met
```

Comment:
Comment might be confusing since STATE_BUFFERING no longer appears - maybe clarify to "Propagate player state to buffering by setting mediaState to loading if these conditions are met" or sth.